util/src/main/java/io/kubernetes/client/Configmap.java

```diff
@@ -0,0 +1,197 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client;
+
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.Configuration;
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.V1ConfigMap;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.util.exception.CreateConfigmapException;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.io.StringReader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import okhttp3.Call;
+import okhttp3.Response;
+
+/** Utility class to create Configmaps */
+public class Configmap {
+
+  private ApiClient apiClient;
+  private CoreV1Api coreClient;
+
+  public Configmap() {
+    this(Configuration.getDefaultApiClient());
+  }
+
+  public Configmap(ApiClient apiClient) {
+    this.apiClient = apiClient;
+    this.coreClient = new CoreV1Api(apiClient);
+  }
+
+  public ApiClient getApiClient() {
+    return apiClient;
+  }
+
+  public void setApiClient(ApiClient apiClient) {
+    this.apiClient = apiClient;
+  }
+
+  /**
+   * Method to create a Configmap from a file. The file should contain key-value pairs
+   *
+   * @param namespace
+   * @param name
+   * @param file
+   * @throws IOException
+   * @throws CreateConfigmapException
+   * @throws ApiException
+   */
+  public void createFromFile(String namespace, String name, Path file)
+      throws IOException, CreateConfigmapException, ApiException {
+    if (Files.isRegularFile(file)) {
+      try (BufferedReader bufferedReader =
+          new BufferedReader(
+              new InputStreamReader(Files.newInputStream(file), StandardCharsets.UTF_8.name()))) {
```

Comment:
Just use `new FileReader(...)` instead of `InputStreamReader(....)`

Reply:
I have changed it.